src/components/VideoDetails/VideoDetails.test.jsx

```diff
@@ -14,7 +14,7 @@ describe('VideoDetails component', () => {
     const renderVideoDetails = () =>
       render(
         <Router history={history}>
-            <VideoDetails getVideosFn={getVideosFn} loadVideoFn={loadVideoFn} />
+          <VideoDetails getVideosFn={getVideosFn} loadVideoFn={loadVideoFn} />
```

Comment:
Was this the reason for adding the defaults?. If this is for unit testing only, you should mock the `getVideos` and `loadVideo` functions. You should never add props or functionality to make your tests work. Rely only on function mocks or mocked data for them, but don't change your implementation to make the test work.